package.json

```diff
@@ -12,15 +12,16 @@
     "postinstall": "node ./scripts/postinstall.js",
     "buildto": "node ./scripts/monorepo/buildTo.js",
     "builddemo": "node ./scripts/monorepo/buildTo.js fabric-website-resources --min",
+    "bundle": "lerna run bundle --stream --",
     "scrub": "node ./scripts/scrub.js",
     "vrtest": "cd apps && cd vr-tests && npm run screener",
     "a11ytest": "cd apps && cd a11y-tests && npm run test",
     "start": "yarn workspace @uifabric/fabric-website-resources start",
     "start-exp": "yarn workspace @uifabric/experiments start",
-    "build": "lerna run build --stream -- --production --lint",
-    "buildci": "lerna run build --stream -- --lint",
+    "build": "lerna run build --stream --",
```

Comment:
Is `--` necessary here?

Reply:
This is in here for now because I'm not sure how else to pass the `--production` flag to `build`. It implicitly worked with `buildci` before because the arg comes after `--` there as well.

Reply:
I've removed stray instances of trailing `--` and added a few `:prod` convenience commands. Funny enough I found in testing that `yarn build -- --production` does not work, but `yarn build -- -- --production` DOES. I've decided to add the `:prod` as convenience commands, which are only slightly less awkward.